localgovdrupal/localgov
# enables admin_role module

Closes #748 

## What does this change?
Enables the new `localgov_admin_role` module, so we have a consistent way of working with an admin role.

## How to test
Install the distro, check that there is now an 'Admin' role and that is set to be the role used for the administrator role.

---
Thanks to [Big Blue Door](https://www.bigbluedoor.net/) for sponsoring my time to work on this.
